src/couch/src/couch_rand.erl

```diff
@@ -14,11 +14,34 @@
 
 
 -export([
+    crypto_to_rand/2,
     uniform/0,
     uniform/1
 ]).
 
 
+%% This function takes the Lo and Hi parameters of crypto_rand as input
+%% and calculates the corresponding N value for couch_rand:uniform/1, and
+%% the offset necessary to add to the latter to generate the same range as
+%% crypto:rand/2.
+%%
+%% crypto:rand_uniform(LoInclusive, HiExclusive) generates a random number N
+%% where LoInclusive =< N < HiExclusive [1]
+%%
+%% rand:uniform(HiInclusive) and random:uniform(HiInclusive) both
+%% generate a random number N where 1 =< X =< HiInclusive [2][3].
+%%
+%% [1] http://erlang.org/doc/man/crypto.html#rand_uniform-2
+%% [2] http://erlang.org/doc/man/rand.html#uniform-1
+%% [3] http://erlang.org/doc/man/random.html#uniform-1
+
+crypto_to_rand(LoInclusive, HiExclusive)
```

Comment:
Do we use this function anywhere? I did a page search in the browser and couldn't find any instances.

Reply:
No, but I used it for the crypto:rand_uniform/2 to rand:uniform/1 conversions, and thought I should "show my work". Happy to remove it, though, since it's really only related to this PR.

Reply:
Yeah let's remove if it is not used